src/main/java/org/sonatype/gradle/plugins/scan/ossindex/OssIndexAuditTask.java

```diff
@@ -70,8 +72,10 @@ public void audit() {
     boolean hasVulnerabilities;
 
     try (OssindexClient ossIndexClient = buildOssIndexClient()) {
-      Set<ResolvedDependency> dependencies =
-          dependenciesFinder.findResolvedDependencies(getProject(), extension.isAllConfigurations());
+      Set<ResolvedDependency> dependencies = getProject().getAllprojects().stream()
+          .flatMap(
+              project -> dependenciesFinder.findResolvedDependencies(project, extension.isAllConfigurations()).stream())
+          .collect(Collectors.toCollection(LinkedHashSet::new));
```

Comment:
```suggestion
      Set<ResolvedDependency> dependencies = getProject().getAllprojects().stream()
          .flatMap(
              project -> dependenciesFinder.findResolvedDependencies(project, extension.isAllConfigurations())
              .stream())
          .collect(Collectors.toCollection(LinkedHashSet::new));
```

Personal Preference -> I feel this way the code can be easier to read

Reply:
> Personal Preference -> I feel this way the code can be easier to read

I actually find it easier to read the way it currently is.